### ISSUE TITLE
💅 useExpect: expect.element() (Vitest Browser Mode) is not recognized as a valid assertion

### ISSUE THREAD

**Issue by maks-oleksyuk** [biomejs/biome#11328]
`lint/nursery/useExpect` reports a test as missing an `expect()` assertion even when it contains valid `expect.element(...)` calls — Vitest Browser Mode's primary locator-assertion API (https://vitest.dev/guide/browser/assertion-api).

Root cause: `is_expect_expression` only recognizes a bare `expect.<member>(...)` call as an assertion when `<member>` is one of `soft`, `poll`, `assertions`, `hasAssertions`: https://github.com/biomejs/biome/blob/main/crates/biome_js_analyze/src/frameworks/playwright.rs#L239

```rust
return matches!(
    member_token.text_trimmed(),
    "soft" | "poll" | "assertions" | "hasAssertions"
);
```

`"element"` is missing from this allow-list, so a test whose only assertions are `expect.element(locator).toBeVisible()` (etc.) is reported as having no assertions at all. A test that mixes `expect.element(...)` with even one `expect.poll(...)` call is not flagged, since `contains_expect_call` short-circuits on the first match — that's what made this easy to miss until a test uses `expect.element()` exclusively.

## Environment information

<details>

```bash
CLI:
  Version:                      2.5.8
  Color support:                true

Platform:
  CPU Architecture:             aarch64
  OS:                           linux

Environment:
  BIOME_DISTRIBUTION:                npm
  JS_RUNTIME_VERSION:           v26.7.0
  JS_RUNTIME_NAME:              node
  NODE_PACKAGE_MANAGER:         npm/11.19.0
```

</details>

## Rule name

useExpect

## Playground link

https://biomejs.dev/playground/?lintRules=useExpect&tab=formatter&pane=Diagnostics&code=aQBtAHAAbwByAHQAIAB7ACAAZQB4AHAAZQBjAHQALAAgAHQAZQBzAHQAIAB9ACAAZgByAG8AbQAgACIAdgBpAHQAZQBzAHQAIgA7AAoACgB0AGUAcwB0ACgAIgB1AHMAZQBzACAAZQB4AHAAZQBjAHQALgBwAG8AbABsACAAbwBuAGwAeQAiACwAIABhAHMAeQBuAGMAIAAoACkAIAA9AD4AIAB7AAoAIAAgAGEAdwBhAGkAdAAgAGUAeABwAGUAYwB0AC4AcABvAGwAbAAoACgAKQAgAD0APgAgADEAKQAuAHQAbwBCAGUAKAAxACkAOwAKAH0AKQA7AAoACgB0AGUAcwB0ACgAIgB1AHMAZQBzACAAZQB4AHAAZQBjAHQALgBlAGwAZQBtAGUAbgB0ACAAbwBuAGwAeQAiACwAIABhAHMAeQBuAGMAIAAoACkAIAA9AD4AIAB7AAoAIAAgAGEAdwBhAGkAdAAgAGUAeABwAGUAYwB0AC4AZQBsAGUAbQBlAG4AdAAoAGQAbwBjAHUAbQBlAG4AdAAuAGIAbwBkAHkAKQAuAHQAbwBCAGUAVgBpAHMAaQBiAGwAZQAoACkAOwAKAH0AKQA7AAoA&language=ts

Verified live: the `expect.poll` test produces no diagnostic; the `expect.element` test produces `lint/nursery/useExpect: Test callback is missing an expect() assertion.`

## Expected result

`expect.element(...)` should be recognized as a valid assertion, the same as `expect.poll(...)` — both are documented Vitest Browser Mode assertion APIs. Suggested fix: add `"element"` to the allow-list at `playwright.rs:239`.

- [x] I agree to follow Biome's Code of Conduct
